src/DataViews/SamplingReportDataView.cpp

```diff
@@ -450,4 +455,56 @@ SampledFunction& SamplingReportDataView::GetSampledFunction(unsigned int row) {
   return functions_[indices_[row]];
 }
 
+ErrorMessageOr<void> SamplingReportDataView::WriteStackEventsToCSVFile(
+    const std::string& file_path) {
+  OUTCOME_TRY(auto fd, orbit_base::OpenFileForWriting(file_path));
+
+  static const std::vector<std::string> kNames{"Thread", "Timestamp (ns)", "Names leaf/foo/main",
+                                               "Addresses leaf_addr/foo_addr/main_addr"};
+  static const std::string kFramesSeparator = "/";
+
+  OUTCOME_TRY(WriteLineToCSV(fd, kNames));
+  const orbit_client_data::CallstackData* callstack_data = sampling_report_->GetCallstackData();
+  ORBIT_CHECK(callstack_data != nullptr);
+
+  const std::vector<orbit_client_data::CallstackEvent> callstack_events =
```

Comment:
So this is essentially this will export all callstacks of the selected thread, right? I fear that this is somehow strange to the user. You select certain functions and select right-click -> export csv. Then the CSV contains "almost" all callstacks. I would expect it contains only the ones that contain the selected functions.

Reply:
Discussed offline, function selection is supported now.  Thanks.